README.md

```diff
@@ -27,6 +27,16 @@ Mitiq can be installed from PyPi via
 pip install mitiq
 ```
 
+Installation from source (within a virtual environment) can be done via
+
+```bash
+git clone https://github.com/unitaryfund/mitiq.git
+cd mitiq/
+python3 -m venv myenv
+source ./myenv/bin/activate
+(myenv) pip install -e .
+```
+
```

Comment:
Let's just link to the development installation page instead of adding duplication. E.g., "See [here](https://mitiq.readthedocs.io/en/latest/contributing.html#development-install) for development installation instructions."